Bitcoin-com/wallet_lib
# JSON Fixes + more customizable options for the library

### What does this PR do?  
_This PR fixes JSON problems, ZEC labels disabled, exception handling, and tests._

### Link to Jira

[GAMES-1210](https://bitcoindotcom.atlassian.net/browse/GAMES-1210)

### Steps for testing  
1. Make sure the tests work on your local machine
2. Make sure the examples stills work.
2. Use the wallet lib in a different application and try to some basic commands.

### Deployment notes
_ZEC labels disabled._

- [ ] Database migration required.  Script: ______
